metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,82 @@
+# Title
+
+Utah Roads
+
+## ID
+
+a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
+
+## Brief Summary
+
+The latest geospatial information for all roads and highways in Utah.
+
+## Summary
+
+Utah Roads is a multi-purpose statewide roads dataset for cartography, routing, and range-based address location. This dataset is also used as the base geometry for deriving the geospatial representation of UDOT's advanced highway linear referencing system ([ALRS](https://gis.utah.gov/products/sgid/transportation/highway-routes-lrs/)) as well as [UGRC's network analysis datasets](https://gis.utah.gov/products/sgid/transportation/).
+
+## Description
+
+This dataset includes information such as the road name, type, speed limit, and infrastructure such as bike lanes. To see specific information about the data model including field names and descriptions, data types, coded value domains, and data model updates, see the [data model spreadsheet](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546) and the corresponding tabs for update notes. The color-coded section titled "Addressing" is primarily maintained by local stewards, whereas the section titled "DOT/FHWA" is primarily maintained by UDOT. UGRC spatially assigns fields in the section titled "Placename/Area", based on current SGID polygon boundaries.
+
+This dataset does not contain USFS roads. For the most current and complete set of USFS roads, please visit the [FSGeodata Clearinghouse](https://data.fs.usda.gov/geodata/edw/datasets.php?dsetCategory=transportation).
```

Comment:
I would remove the sentence, "this dataset does not contain usfs roads". It does contain forest service roads, however, we recommend folks use the usfs source data if that is their focus.

Reply:
Oh, I see. Steve said it didn't have the USFS roads. I'll go and put it back in!

Reply:
Yeah I missed a word. It does not contain every usfs road.